### PR TITLE
*: add description about the binary plan

### PR DESCRIPTION
**Pull request by time-and-fate**
### What is changed, added or deleted? (Required)

This PR adds a description of the binary plan, which is to support the visual plan feature on the dashboard side.

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [ ] v6.2 (TiDB 6.2 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s): https://github.com/pingcap/tidb/pull/35698

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: identify-slow-queries.md [hunk 1]
```diff
@@ -67,6 +67,7 @@ Slow Query 基础信息：
 * `Succ`：表示语句是否执行成功。
 * `Backoff_time`：表示语句遇到需要重试的错误时在重试前等待的时间。常见的需要重试的错误有以下几种：遇到了 lock、Region 分裂、`tikv server is busy`。
 * `Plan`：表示语句的执行计划，用 `select tidb_decode_plan('xxx...')` SQL 语句可以解析出具体的执行计划。
+* `Binary_plan`: 表示以二进制格式编码后的语句的执行计划，用 `select tidb_decode_binary_plan('xxx...')` SQL 语句可以解析出具体的执行计划。传递的信息和 `Plan` 字段基本相同，但是解析出的执行计划的格式会和 `Plan` 字段不同。
```
Comment (shichun-0415) [inline identify-slow-queries.md:70]:
```suggestion
* `Binary_plan`：表示以二进制格式编码后的语句的执行计划，用 `select tidb_decode_binary_plan('xxx...')` SQL 语句可以解析出具体的执行计划。传递的信息和 `Plan` 字段基本相同，但是解析出的执行计划的格式会和 `Plan` 字段不同。
```

File: statement-summary-tables.md [hunk 2]
```diff
@@ -245,6 +245,7 @@ SQL 的基础信息：
 - `SAMPLE_USER`：执行这类 SQL 的用户名，多个用户名只取其中一个
 - `PLAN_DIGEST`：执行计划的 digest
 - `PLAN`：原执行计划，多条语句只取其中一条的执行计划
+- `BINARY_PLAN`: 以二进制格式编码后的原执行计划，多条语句只取其中一条的执行计划。用 `select tidb_decode_binary_plan('xxx...')` SQL 语句可以解析出具体的执行计划。
```
Comment (shichun-0415) [inline statement-summary-tables.md:248]:
```suggestion
- `BINARY_PLAN`：以二进制格式编码后的原执行计划，存在多条语句时，只取其中一条语句的执行计划。用 `select tidb_decode_binary_plan('xxx...')` SQL 语句可以解析出具体的执行计划。
```

**Review by shichun-0415 — APPROVED** [review]

**Review by AilinKid — APPROVED** [review]
LGTM

File: system-variables.md [hunk 3]
```diff
@@ -1066,6 +1066,14 @@ v5.0 后，用户仍可以单独修改以上系统变量（会有废弃警告）
 - 这个变量用于改变 TiDB server 上执行的语句的默认优先级。例如，你可以通过设置该变量来确保正在执行 OLAP 查询的用户优先级低于正在执行 OLTP 查询的用户。
 - 可设置为 `NO_PRIORITY`、`LOW_PRIORITY`、`DELAYED` 或 `HIGH_PRIORITY`。
 
+### `tidb_generate_binary_plan` <span class="version-mark">从 v6.2 版本开始引入</span>
```
Comment (ran-huang) [inline system-variables.md:1069]:
```suggestion
### `tidb_generate_binary_plan` <span class="version-mark">从 v6.2.0 版本开始引入</span>
```

File: system-variables.md [hunk 4]
```diff
@@ -1066,6 +1066,14 @@ v5.0 后，用户仍可以单独修改以上系统变量（会有废弃警告）
...
+
+- 作用域：GLOBAL
+- 是否持久化到集群：是
+- 默认值：`ON`
+- 这个变量用于指定是否在 slow log 和 statement summary 里包含以二进制格式编码的执行计划。TiDB Dashboard 中的图形化执行计划依赖该信息。
```
Comment (ran-huang) [inline system-variables.md:1074]: *依赖* 是一个比较模糊的表达，能否尽量准确地说明如何依赖？例如（随便写的，不知道是否正确）：
```suggestion
- 这个变量用于指定是否在 slow log 和 statement summary 里包含以二进制格式编码的执行计划。开启该变量后，可以在 TiDB Dashboard 中查看图形化执行计划。
```

Comment (time-and-fate) [reply system-variables.md:1074]: 是的，是这个意思。
图形化执行计划就是基于 binary plan 实现的。只有这个变量打开的时候，dashboard 才能展示图形化执行计划。

Comment (time-and-fate) [reply system-variables.md:1074]: 想了下，表达的意思稍微有点差异。
> 只有在该变量开启时产生的查询，才能在 TiDB Dashboard 中展示对应的图形化执行计划。

这个意思准确一点。

Comment (shichun-0415) [reply system-variables.md:1074]: “只有...才”的描述感觉不够突出该变量的强大，这样描述可以吗：
< 开启该变量后，即可在 TiDB Dashboard 中查看新查询的图形化执行计划。

Comment (time-and-fate) [reply system-variables.md]: 嗯，感觉可以。已更新。

**Review by shichun-0415 — APPROVED** [review]

**Review by ran-huang — APPROVED** [review]

**Pull request merged**